source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/mdp/rewards.py

```diff
@@ -104,3 +104,21 @@ def track_ang_vel_z_world_exp(
     asset = env.scene[asset_cfg.name]
     ang_vel_error = torch.square(env.command_manager.get_command(command_name)[:, 2] - asset.data.root_ang_vel_w[:, 2])
     return torch.exp(-ang_vel_error / std**2)
+
+
+def stand_still_error(env, command_name: str, asset_cfg: SceneEntityCfg = SceneEntityCfg("robot")) -> torch.Tensor:
+    """Penalize offsets from the default joint positions when the command is very small."""
+    asset = env.scene[asset_cfg.name]
+    command = env.command_manager.get_command(command_name)
+    # Penalize motion at zero commands
+    offset = asset.data.default_joint_pos[:, asset_cfg.joint_ids] - asset.data.joint_pos[:, asset_cfg.joint_ids]
+    return torch.sum(torch.abs(offset), dim=1) * (torch.norm(command[:, :2], dim=1) < 0.06)
+
+
+def no_jumps(env, sensor_cfg: SceneEntityCfg) -> torch.Tensor:
```

Comment:
We already have a MDP term called: undesired_contacts which does the same operation?

https://isaac-sim.github.io/IsaacLab/main/source/api/lab/isaaclab.envs.mdp.html#isaaclab.envs.mdp.rewards.undesired_contacts

Reply:
Ah okay. Then I'd say we should make another term in the `mdp.rewards` called `desired_contacts`? That might be useful for some other tasks in the future too